docs/docs/content-and-data.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Sourcing Content and Data
+title: Carregando Conteúdo e Dados
 ---
 
-A core feature of Gatsby is its ability to **load data from anywhere**. This is part of what makes Gatsby more powerful than static site generators that are limited to only loading content from Markdown files.
+O principal benefício em usar a técnica **"dados de qualquer lugar"**, é que ela permite que times consigam gerenciar seu conteúdo no backend que preferirem. Isso é parte do que torna o Gatsby mais poderoso que os geradores de sites estáticos que são limitados a carregar conteúdo apenas de arquivos *Markdown*.
```

Comment:
```suggestion
O principal benefício em usar a técnica **"dados de qualquer lugar"**. Isso é parte do que torna o Gatsby mais poderoso que os geradores de sites estáticos que são limitados a carregar conteúdo apenas de arquivos *Markdown*.
```

Eu não sei se foi escolha tua, mas não entendi porque tu duplica a tradução de "A core benefit of this “data from anywhere” approach is that it allows teams to manage their content in nearly any [backend](/docs/glossary/#backend) they prefer.", se tu achasse válido, acho que deixar apenas a parte de baixo falar sobre o backend talvez seja melhor.

Reply:
Passou batido durante o processo de tradução. Aceita a sugestão.